src/components/mfa/MultiFactorAuthentication.tsx

```diff
@@ -0,0 +1,244 @@
+ * ```js
+ * import * as React from 'react'
+ * import {
+ *   AppUserMfaProvider,
+ *   MultiFactorAuthentication,
+ * } from '@oneblink/apps-react'
+ *
+ * function Component() {
+ *   return <MultiFactorAuthentication />
+ * }
+ *
+ * function App() {
+ *   return (
+ *     <AppUserMfaProvider>
+ *       <Component />
+ *     </AppUserMfaProvider>
+ *   )
+ * }
+ *
+ * const root = document.getElementById('root')
+ * if (root) {
+ *   ReactDOM.render(<App />, root)
+ * }
+ * ```
+ *
+ * @param props
+ * @returns
+ * @group Components
+ */
+export default function MultiFactorAuthentication({
+  ssoSetupUrl,
+  isExternalIdentityProviderUser,
+}: Props) {
+  const {
+    loadingError,
+    setupError,
+    isLoading,
+    isMfaEnabled,
+    isDisablingMfa,
+    isSettingUpMfa,
+    mfaSetup,
+    loadAppUserMfa,
+    beginMfaSetup,
+    cancelMfaSetup,
+    completeMfaSetup,
+    clearMfaSetupError,
+    beginDisablingMfa,
+    completeDisablingMfa,
+    cancelDisablingMfa,
+  } = useAppUserMfa()
+
+  return (
+    <Grid item xs={true} lg={8}>
+      <Box padding={3}>
+        <Paper>
+          <Box padding={3}>
+            <Grid container spacing={2} alignItems="center">
+              <Grid item xs>
+                <Typography variant="h4" fontWeight="light">
+                  Multi Factor Authentication{' '}
+                  {isExternalIdentityProviderUser ? undefined : isLoading ? (
+                    <Box padding={3}>
+                      <Grid container justifyContent="center">
+                        <CircularProgress />
+                      </Grid>
+                    </Box>
+                  ) : loadingError ? (
+                    <div>
+                      <ErrorMessage
+                        title="Error Loading Multi Factor Authentication Configuration"
+                        onTryAgain={loadAppUserMfa}
+                      >
+                        {loadingError.message}
+                      </ErrorMessage>
+                    </div>
+                  ) : isMfaEnabled ? (
+                    <Chip
+                      label="Enabled"
+                      icon={
+                        <MaterialIcon color="success">
+                          verified_user
+                        </MaterialIcon>
+                      }
+                    />
+                  ) : (
+                    <Chip
+                      label="Disabled"
+                      icon={
+                        <MaterialIcon color="warning">
+                          remove_moderator
+                        </MaterialIcon>
+                      }
+                    />
+                  )}
+                </Typography>
+                <Box marginY={1}>
+                  <Divider />
+                </Box>
+                <Typography variant="body2" paragraph>
+                  Multi factor authentication (MFA), also known as two factor
+                  authentication (2FA), is a best practice that requires a
```

Comment:
not quite true ?

https://www.incognia.com/the-authentication-reference/what-are-the-key-differences-between-2fa-and-mfa#:~:text=MFA%20vs%202FA,all%20MFA%20is%20a%202FA.